api/v1/serializers/instance_serializer.py

```diff
@@ -51,6 +52,14 @@ def __init__(self, *args, **kwargs):
         self.request_user = user
         super(InstanceSerializer, self).__init__(*args, **kwargs)
 
+    def get_size(self, obj):
+        from api.v2.serializers.summaries import (
```

Comment:
PEP8 recommends imports only at the top of the file, but I can see that there are already function-level imports elsewhere in this code.

From what I have read there may be a slight performance cost to importing inside the function, but probably only on the first call to the function (after that the module should be cached).

In discussions about this on StackOverflow there were a few notes that some people do import within the function while developing, until they have some stable code and know what they need and where it is used, then they move all the imports to the top to comply with PEP8.

Mostly my thought is that if we are going to violate PEP8 there should be a reason, although the reason could be simple like "consistency with existing code" etc.